products-service/serverless.ts

```diff
@@ -23,11 +25,16 @@ const serverlessConfiguration: AWS = {
     },
     environment: {
       AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      PG_HOST: '',
```

Comment:
Don't you want to setup it here too?

Reply:
Not sure what you've meant.

Reply:
Just remember from our example from lesson that db connection config is additionally setup in serverless file too

Reply:
Do you mean .env file?